backends/advanced/src/advanced_omi_backend/services/plugin_service.py

```
@@ -163,6 +163,284 @@ def set_plugin_router(router: PluginRouter) -> None:
+    """
+    # Load plugin config.yml
+    try:
+        import advanced_omi_backend.plugins
+        plugins_dir = Path(advanced_omi_backend.plugins.__file__).parent
+        plugin_config_path = plugins_dir / plugin_id / "config.yml"
+
+        config_dict = {}
+        if plugin_config_path.exists():
+            with open(plugin_config_path, 'r') as f:
+                config_dict = yaml.safe_load(f) or {}
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)